index.html

```diff
@@ -0,0 +1,286 @@
+                       </li>
+                       <li>
+                           <a href="#"><img src="images/linkedInIcon.png" alt=""></a>
+                       </li>
+                       <li>
+                           <a href="#"><img src="images/mIcon.svg" alt=""></a>
+                       </li>
+                       <li>
+                           <a href="#"><img src="images/gitHubIcon.svg" alt=""></a>
+                       </li>
+                       <li>
+                           <a href="#"><img src="images/peaceIcon.svg" alt=""></a>
+                       </li>
+                   </ul>
+               </div>
+               <br>
+               <button type="button"><img src="images/Getmyresume.png" alt=""></button>
+               </div>
+               <div class="langs">
+                <ul>
+                   <li class="lang-icon"><img src="images/Languages.png" alt=""></li>
+                   <li class="greaterthan"><a href="">&#86;</a></li>
+                </ul>
+               </div>
+               <div class="frameworks">
+                   <ul>
+                       <li><img src="images/Frame9.png" alt=""></li>
+                       <li><img src="images/Frame10.png" alt=""></li>
+                       <li><img src="images/Frame11.png" alt=""></li>
+                   </ul>
+               </div>
+               <div class="langs">
+                <ul>
+                   <li class="lang-icon"><img src="images/Frameworks1.png" alt=""></li>
+                   <li class="lang-text"><a href="">&#62;</a></li>
+                </ul>
+               </div>
+               <p><img src="images/SeparatorBottom.png" alt=""></p>
+               <div class="langs">
+                <ul>
+                   <li class="lang-icon"><img src="images/Skills1.png" alt=""></li>
+                   <li class="skills-text lang-text"><a href="">&#62;</a></li>
+                </ul>
+               </div>
+               <p><img src="images/SeparatorBottom.png" alt=""></p>
+           </div>       
+           </section>
+           <section class="works-2">
+            <div class="headline1">
+               <div class="headline-box">
+                 <h2 class="mainheader">
+                     Contact Me</h2>
+                 <p class="details-2">
+                   <img src="images/PrimaryText3.png" alt="">
+                 </p>
+               </div>
+               <div class="form-input">
+                <form action="#">
+                    <div class="container">
+                      <input type="text" placeholder="" name="Name" id="name">                
+
+                      <input type="text" placeholder="" name="email" id="email">
+                  
+                      <textarea class="textbox" placeholder="" name="comments" id="comments"></textarea>                                      
+                      
+                    </div>           
```

Comment:
It would be good to put related code close together unless there is a reason for the spaces you left....